specification/web/resource-manager/Microsoft.CertificateRegistration/stable/2024-11-01/CertificateOrdersDiagnostics.json

```diff
@@ -0,0 +1,190 @@
+      "get": {
+        "tags": [
+          "CertificateOrdersDiagnostics"
+        ],
+        "summary": "Microsoft.CertificateRegistration call to get a detector response from App Lens.",
+        "description": "Description for Microsoft.CertificateRegistration call to get a detector response from App Lens.",
+        "operationId": "CertificateOrdersDiagnostics_GetAppServiceCertificateOrderDetectorResponse",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "name": "certificateOrderName",
+            "in": "path",
+            "description": "The certificate order name for which the response is needed.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "detectorName",
+            "in": "path",
+            "description": "The detector name which needs to be run.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "startTime",
+            "in": "query",
+            "description": "The start time for detector response.",
+            "type": "string",
+            "format": "date-time"
+          },
+          {
+            "name": "endTime",
+            "in": "query",
+            "description": "The end time for the detector response.",
+            "type": "string",
+            "format": "date-time"
+          },
+          {
+            "name": "timeGrain",
+            "in": "query",
+            "description": "The time grain for the detector response.",
+            "type": "string",
+            "pattern": "PT[1-9][0-9]+[SMH]"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "../../../Microsoft.Web/stable/2024-11-01/CommonDefinitions.json#/definitions/DetectorResponse"
+            }
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "../../../Microsoft.Web/stable/2024-11-01/CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get app service certificate order detector response": {
+            "$ref": "./examples/Diagnostics_GetAppServiceCertificateOrderDetectorResponse.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {},
+  "parameters": {
+    "subscriptionIdParameter": {
+      "name": "subscriptionId",
+      "in": "path",
+      "description": "Your Azure subscription ID. This is a GUID-formatted string (e.g. 00000000-0000-0000-0000-000000000000).",
+      "required": true,
+      "type": "string"
+    },
+    "resourceGroupNameParameter": {
```

Comment:
>resourceGroupNameParameter [](http://example.com/codeflow?start=5&length=26)

Use the definition from v6 common types instead of replicating it here.